From: Danila Tikhonov <danila@jiaxyga.com>
Date: Fri, 21 Feb 2025 22:24:59 +0300
Subject: FROMLIST: dt-bindings: interconnect: qcom-bwmon: Add the SM7150
 compatible

Document the compatibles used to describe the bwmons present on the
SM7150 platform.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
---
 Documentation/devicetree/bindings/interconnect/qcom,msm8998-bwmon.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/interconnect/qcom,msm8998-bwmon.yaml b/Documentation/devicetree/bindings/interconnect/qcom,msm8998-bwmon.yaml
index 256de14..2ced6f5 100644
--- a/Documentation/devicetree/bindings/interconnect/qcom,msm8998-bwmon.yaml
+++ b/Documentation/devicetree/bindings/interconnect/qcom,msm8998-bwmon.yaml
@@ -35,6 +35,7 @@ properties:
               - qcom,sdm845-cpu-bwmon
               - qcom,sm6115-cpu-bwmon
               - qcom,sm6350-llcc-bwmon
+              - qcom,sm7150-cpu-bwmon
               - qcom,sm8250-cpu-bwmon
               - qcom,sm8550-cpu-bwmon
               - qcom,sm8650-cpu-bwmon
@@ -49,6 +50,7 @@ properties:
               - qcom,sc7180-llcc-bwmon
               - qcom,sc8280xp-llcc-bwmon
               - qcom,sm6350-cpu-bwmon
+              - qcom,sm7150-llcc-bwmon
               - qcom,sm8250-llcc-bwmon
               - qcom,sm8550-llcc-bwmon
               - qcom,sm8650-llcc-bwmon
